### ISSUE TITLE
Feature: Time travel debugging with Python 

### ISSUE THREAD

**Issue by ChillarAnand** [reloadware/reloadium#96]
Any plans on supporting time travel debugging with python?

**Comment by dkrystki** [comment]
Yes, so far we've added dropping call stack frames functionality and we're planning to implement more time travel debugging features in the future.
